var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -67,6 +63,40 @@ class CbtfArgonavis(Package):
 
     parallel = False
 
+    build_directory = 'build_cbtf_argonavis'
+
+    def build_type(self):
+        return 'None'
+
+    def cmake_args(self):
+        spec = self.spec
+
+        # Look for package installation information in the cbtf and cbtf-krell
+        # prefixes
+        cmake_prefix_path = join_path(
+            spec['cbtf'].prefix) + ':' + join_path(spec['cbtf-krell'].prefix)
+
+        cmake_args = [
+            '-DCMAKE_PREFIX_PATH=%s' % cmake_prefix_path,
```

Comment:
Ok, but I don't know how the build knows what to provide, unless it includes all the depends_on packages in CMAKE_PREFIX_PATH.